src/app/[locale]/components/FormProgress.tsx

```diff
@@ -0,0 +1,71 @@
+'use client'
+
+import React, { useEffect, useState } from 'react'
+import { Button, Heading } from '@utrecht/component-library-react'
+import { useTranslations } from 'next-intl'
+import { useFormStore } from '@/store/form_store'
+import { stepToPath, usePathname, useRouter } from '@/routing/navigation'
+import { FaChevronLeft } from 'react-icons/fa'
+import { FormStep } from '@/types/form'
+import { getCurrentStep, getPreviousStepPath } from '@/lib/utils/stepper'
+
+const FormProgress = () => {
+  const t = useTranslations('stepper')
+  const { resetForm } = useFormStore()
+  const router = useRouter()
+  const pathname = usePathname()
+  const step = getCurrentStep(pathname)
+
+  const [percentage, setPercentage] = useState<number>(1)
```

Comment:
Stel dat we percentage standaard op 25 zetten? Dan heb je volgens mij niet zo'n soort van "layout shift" in de progress bar zelf bij first load?

Reply:
het is nu toch ook standaard 25%? default waarde is 1 --> 1 / 4 * 100 = 25

Reply:
in het begin was het 0 en toen heb ik dat op basis van jouw comment naar 1 veranderd ;)